api/v1/api.go

```diff
@@ -305,7 +308,11 @@ func (api *API) listAlerts(w http.ResponseWriter, r *http.Request) {
 	defer alerts.Close()
 
 	api.mtx.RLock()
+	alertCount := 0
 	for a := range alerts.Next() {
+		if api.apiLimit != nil && api.apiLimit.MaxAlertsCount > 0 && api.apiLimit.MaxAlertsCount <= alertCount {
```

Comment:
Wonder if you can just use the length of res for the check instead of having another new variable

Reply:
fixed